### PR TITLE
Update Simplified Chinese translations

### PR DESCRIPTION
**Pull request by notname000**

### CODE REVIEW

File: qterminal/qterminal_zh_CN.ts [hunk 1]
```diff
@@ -530,17 +530,17 @@
...
         <location filename="../../forms/propertiesdialog.ui" line="558"/>
         <source>Default $TERM</source>
-        <translation type="unfinished"></translation>
+        <translation>默认 $TERM</translation>
     </message>
     <message>
         <location filename="../../forms/propertiesdialog.ui" line="572"/>
         <source>xterm</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (yan12125) [inline qterminal/qterminal_zh_CN.ts:538]: Forgotten?

File: qterminal/qterminal_zh_CN.ts [hunk 2]
```diff
@@ -530,17 +530,17 @@
...
     </message>
     <message>
         <location filename="../../forms/propertiesdialog.ui" line="577"/>
         <source>xterm-256color</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (yan12125) [inline qterminal/qterminal_zh_CN.ts:543]: ditto

File: qterminal/qterminal_zh_CN.ts [hunk 3]
```diff
@@ -750,7 +750,7 @@
     <message>
         <location filename="../../tabwidget.cpp" line="208"/>
         <source>Select new tab title color</source>
-        <translation type="unfinished"></translation>
+        <translation>选择新标签的标题颜色</translation>
```
Comment (yan12125) [inline qterminal/qterminal_zh_CN.ts:753]: Well, the English text for this item is misleading. The corresponding Chinese should be something like "选择新的标签标题颜色"

File: qterminal/qterminal_zh_CN.ts (hunk 1 above)
Comment (notname000) [reply qterminal/qterminal_zh_CN.ts:538]: It needn't translation.

File: qterminal/qterminal_zh_CN.ts (hunk 2 above)
Comment (notname000) [reply qterminal/qterminal_zh_CN.ts:543]: It needn't translation.

File: qterminal/qterminal_zh_CN.ts (hunk 1 above)
Comment (yan12125) [reply qterminal/qterminal_zh_CN.ts:538]: Sorry not noticing that. You're right.

**Pull request closed without merging**